### PR TITLE
Spring global exception and validator solution

### PR DESCRIPTION
**Pull request by keda-vlad**

### CODE REVIEW

File: src/main/java/mate/academy/bookstore/dto/CreateBookRequestDto.java [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
     private String title;
+    @NotNull
     private String author;
+    @NotNull
     private String isbn;
```
Comment (padu12) [inline src/main/java/mate/academy/bookstore/dto/CreateBookRequestDto.java]: we can add `@ISBN` to check its correctness

```suggestion
    @NotNull
    @ISBN
    private String isbn;
```

**Review by padu12 — APPROVED** [review]
Amazing!

File: pom.xml [hunk 2]
```diff
@@ -34,6 +34,10 @@
             <artifactId>liquibase-core</artifactId>
             <version>${liquibase.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.hibernate.validator</groupId>
```
Comment (vovasalyha) [inline pom.xml]: You can add spring boot validation starter instead of concrete implementation.

**Review by vovasalyha — APPROVED** [review]
GJ

**Pull request merged**